cfbs/commands.py

```diff
@@ -64,6 +65,39 @@ def __init__(self, message):
 FIRST_ARG_SLIST = lambda args, _: ", ".join("'%s'" % module for module in args[0])
 
 
+def get_command_names():
+    # TODO: Consider making a decorator for command functions to make this
+    #       less error prone.
+    names = (
+        "add",
+        "build",
+        "clean",
+        "download",
+        "get-input",
+        "help",
+        "info",
+        "init",
+        "input",
+        "install",
+        "pretty",
+        "remove",
+        "search",
+        "set-input",
+        "show",
+        "status",
+        "update",
+        "validate",
+    )
+    defined = dir(sys.modules[__name__])
+    for command in names:
+        function_name = command.replace("-", "_") + "_command"
+        assert function_name in defined, "'%s' function not found in %s" % (
+            function_name,
+            defined,
+        )
```

Comment:
Do we really need this check?

Reply:
It was very helpful while I was refactoring, but we can remove it now (I already did in a later commit, will squash).